src/content/blog/2024/reflections-on-nextflow-mentorship.md

```diff
@@ -0,0 +1,78 @@
+
+<div style="margin-top: 2rem; margin-bottom: 2rem;">
+    <img src="/img/blog-2024-04-10-img1b.png" alt="Review comment in GitHub" />
+</div>
+
+For my final project in the mentorship program, I successfully ported a complete pipeline from Bash to Nextflow. This was a learning experience that allowed me to explore a diverse range of skills, such as modularizing content, understanding how crucial the meta map is, and creating Docker container images for software. This process not only enhanced my proficiency in Nextflow but also allowed me to interact with and contribute to related projects like Bioconda and BioContainers.
+
+# Life after the mentorship
+
+With the skills I acquired during the mentorship as a mentee, I proposed and successfully implemented a custom solution in Nextflow for a precision medicine start-up I worked at the time that could sequentially do several diagnostics and consumer-genetics applications in the cloud, resulting in substantial cost savings and increasing flexibility for the company.
+Beyond my immediate projects, I joined a group actively developing an open-source Nextflow pipeline for genetic imputation. This project allowed me to be in close contact with members of the nf-core community working on similar projects, adding new tools to this pipeline, giving and receiving feedback, and continuing to improve my overall Nextflow skills while also contributing to the broader bioinformatics community. You can learn more about this project with the fantastic talk by Louis Le Nézet at Nextflow Summit 2023 Nextflow Summit 2023 - Louis Le Nézet
+Finally, I was honored to become a Nextflow ambassador. The program’s goal is to extend the awareness of Nextflow around the world while also building a supportive community. In particular, the South American community is underrepresented, so I serve as a point of contact for any institution or newcomer who wants to implement pipelines with Nextflow.
+As part of this program, I was invited to speak at the second Chilean Congress of Bioinformatics, where I gave a talk about how Nextflow and nf-core can support scaling bioinformatics projects in the cloud. It was incredibly rewarding to introduce Nextflow to a community for the first time and witness the genuine enthusiasm it sparks among students and attendees for the potential in their research projects.
+
+<div style="margin-top: 2rem; margin-bottom: 2rem;">
+    <img src="/img/blog-2024-04-10-img1c.png" alt="Second Chilean Congress of Bioinformatics" />
+</div>
+
+# What’s next?
+
+The comprehensive skill set acquired in my journey resonated positively and allowed me to join the ZS Discovery Team. This organization accelerates transformation in research and early development with direct contribution to impactful bioinformatics projects with a globally distributed, multidisciplinary talented team.
```

Comment:
"resonated positively" is a slightly odd phrasing, would suggest "proved to be incredibly valuable for my professional development" instead. Also "allowed me to join the ZS Discovery Team" feels too modest, would suggest to add "as a Senior Bioinformatician" which is very cool by the way.